libs/theme/src/defaultTheme.tsx

```diff
@@ -1253,6 +1253,7 @@ const defaultComponentConfig = {
     stepContainer: {
       master: "flex items-center space-x-4",
       padding: "py-4 px-6",
+      margin: "mr-1",
```

Comment:
Why this margin? It looks fine without it.

Reply:
It is required to fill in the width `w-1` of the separator. Small difference but provides better symmetry.